backend/src/main/java/com/ody/route/config/RouteConfig.java

```diff
@@ -26,6 +27,7 @@ public class RouteConfig {
     private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(30);
 
     private final RouteClientProperties properties;
+    private final ObjectMapper objectMapper;
```

Comment:
ObjectMapper를 customize하고 자동 주입받는 것이라면 필드로 가지고 있지 말고, RestClientBuilder 빈 등록에서 파라미터로 받아서 등록해도 좋을 것 같아요

```suggestion
    @Bean
    public RestClient.Builder builder(ObjectMapper objectMapper) {
        return RestClient.builder()
                .requestFactory(new BufferingClientHttpRequestFactory(clientHttpRequestFactory()))
                .requestInterceptor(new RouteClientLoggingInterceptor(objectMapper));
    }
```

Reply:
반영 완료!